fond.css

```diff
@@ -0,0 +1,48 @@
+body{
+    background-image: url("assets/desktop-wallpaper-dark-grey-background-dark.jpeg");
+}
+
+.my-bread-crumb{
+    background-color: black;
+}
+
+.ligne{
+    border: solid 2px white;
+    
+}
+
+
+.lettre{
+    height:20rem;
+    width:fit-content;
+   
+}
+
+
+.card{
+    margin-left: 30px;
+    margin-bottom: 10px;
+    box-shadow: -10px 0px 10px grey;
+}
+
+.card-body{
+background-color: white;
```

Comment:
Indentation is not good. Try to format your code (Right Click -> Format Code) to fix it.